### ISSUE TITLE
Add tags latest / stable?

### ISSUE THREAD

**Issue by pwFoo** [linuxkit/linuxkit#1823]
All images build with a hash as docker image tag. So each image have to be replaced for an update. Additional docker image tags latest / testing and stable would be nice for easier update linuxkit builds?

**Comment by justincormack** [comment]
I don't think we will do any tags until we do a release. We havent quite decided how often to do releases or when to do the first one, but we should decide soon.

**Comment by pwFoo** [comment]
Ok, but point the tag latest to the current / latest hash would help to upgrade a linuxkit builds. I changed all the image hashes to get the latest docker-ce image. So instead of stable just add a latest / testing / current "alias" tag?

**Comment by justincormack** [comment]
There are a lot of problems with `latest`, in particular it is very hard to know what anyone is running as it depends when they pulled the images, and we don't yet have tests for all the images so we don't have a good handle on what is working. We do plan to introduce a templating system that should make this easier to manage.
